assets/src/edit-story/components/helpCenter/constants.js

```diff
@@ -87,7 +72,7 @@ export const TIPS = {
     title: __('Add links to design elements', 'web-stories'),
     description: [
       __(
-        'Go to the Design tab. Scroll down to Links and enter a URL. To remove a link, select the “X” to the right of URL box.',
+        'Go to the <strong>Links</strong> section under the Design tab and enter a URL. To remove a link, select the "X"<screenreader> (Clear)</screenreader> button.',
```

Comment:
Where's the `TranslateWithMarkup` for the `<strong>` here?

Seems weird that it's not used here, feels disconnected